src/models/Lessons.js

```diff
@@ -127,7 +127,7 @@ export function getByMonth(number) {
   return Lessons.sequalize.query('SELECT * FROM lessons WHERE EXTRACT(MONTH FROM date) = :number',
     { replacements: { number } },
     { type: Lessons.sequalize.QueryTypes.SELECT })
-    .then(lessons => lessons[0])
+    .then(lessons => (lessons.length ? lessons[0] : Promise.reject()))
```

Comment:
You should add rejection reason (not only here)

Reply:
Done.